recipes/r-fastverse/meta.yaml

```
@@ -0,0 +1,91 @@
+{% set version = '0.2.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-fastverse
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/fastverse_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/fastverse/fastverse_{{ version }}.tar.gz
+  sha256: 473c4c7ddb0277906d39f729a5d1057af64b66468a8a2ba782f1a7d06fa4e9ca
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+
+requirements:
+  build:
+    - {{ compiler('c') }}          # [not win]
+    - {{ compiler('m2w64_c') }}    # [win]
+    - {{ posix }}zip               # [win]
+    - cross-r-base {{ r_base }}    # [build_platform != target_platform]
+  host:
+    - r-base
+    - r-collapse
+    - r-data.table
+    - r-fst
+    - r-kit
+    - r-magrittr
+    - r-matrixstats
+  run:
+    - r-base
+    - {{ native }}gcc-libs         # [win]
+    - r-collapse
+    - r-data.table
+    - r-fst
+    - r-kit
+    - r-magrittr
+    - r-matrixstats
+
+test:
+  commands:
+    - $R -e "library('fastverse')"           # [not win]
+    - "\"%R%\" -e \"library('fastverse')\""  # [win]
+
+about:
+  home: https://sebkrantz.github.io/fastverse/, https://github.com/SebKrantz/fastverse
+  license: GPL-3
```

According to CRAN metadata `# License: GPL-3`:
```suggestion
  license: GPL-3.0-only
```